openverse_catalog/dags/commoncrawl/commoncrawl_scripts/commoncrawl_s3_syncer/SyncImageProviders.py

```diff
@@ -17,13 +18,12 @@
 def getCrawlIndex(_param):
 
     if not _param:  # get the most recent index from common crawl
-        bucket = "commoncrawl"
         s3 = boto3.client("s3", config=Config(signature_version=UNSIGNED))
 
         # verify bucket
         contents = []
         prefix = "cc-index/collections/CC-MAIN-"
```

Comment:
I have no idea what this prefix is for, but is it still correct?

Reply:
I'm really not sure :sweat_smile: I assume there will be a whole host of things we'll need to change once we actually try and get this going!